proposals/lip-0007.md

```diff
@@ -51,21 +51,21 @@ The result being delegates and node operators are always given the correct infor
 
 ## Specification
 
-### A. Protocol Versioning
+### A. Network Versioning
 
-The protocol version will be denoted by two digits, **H.S**. The first digit, **H**, depends on the number of hard forks. More precisely, **H** is incremented with every hard fork. **S** represents the number of soft forks since the last hard fork.  We define the initial protocol version 1.0 to be the one that was implemented by Lisk Core 1.0.0.
+The network version will be denoted by two digits, **H.S**. The first digit, **H**, depends on the number of hard forks. More precisely, **H** is incremented with every hard fork. **S** represents the number of soft forks since the last hard fork.  We define the initial network version 1.0 to be the one that was implemented by Lisk Core 1.0.0.
```

Comment:
```suggestion
The network version will be denoted by two digits, **H.S**. The first digit, **H**, depends on the number of hard forks. More precisely, **H** is incremented with every hard fork. **S** represents the number of soft forks since the last hard fork.  The default initial network version of any blockchain created with the Lisk SDK is 1.0.
```